make_crimes_by_street_range.R

```diff
@@ -0,0 +1,110 @@
+library(tidyverse)
+library(sf)
+library(lubridate)
+source("street_range_functions.R")
+
+# download data 
+options(timeout = 200)
+
+download.file(
+  "https://data.cincinnati-oh.gov/api/views/k59e-2pvf/rows.csv?accessType=DOWNLOAD",
+  destfile = "data/crime_incidents_rawdata.csv"
+)
+
+# read in data 
+d <- read_csv("data/crime_incidents_rawdata.csv",
```

Comment:
You can use `readr::read_csv` to read the file directly from the web rather than writing to the working directory of the R process.